gamemodes/terrortown/gamemode/client/cl_marker_vision_data.lua

```diff
@@ -133,6 +133,20 @@ function MARKER_VISION_DATA:SetTitle(text, color, inline_icons)
     }
 end
 
+---
+-- Sets the subtitle of the specific targetID element
```

Comment:
```suggestion
-- Sets the subtitle of the specific radar vision element
```

radar vision is what its called in all other functions

Reply:
yeah, copy pasta, best pasta, oof